### PR TITLE
CHE-2520: add stack testing widget

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Add possibility to test stack with temporary workspace.

#2520
selection_025

selection_022

Signed-off-by: Oleksii Orel <oorel@codenvy.com>

### CODE REVIEW

**Comment by slemeur** [comment]
@olexii4 Sorry: could we change the text on the primary button to be "Test Workspace" ?

File: dashboard/src/app/projects/create-project/samples/create-project-samples.controller.ts [hunk 1]
```diff
@@ -9,110 +9,76 @@
...
 'use strict';
+import {CheAPI} from '../../../../components/api/che-api.factory';
+import {CreateProjectController} from '../create-project.controller';
 
 /**
  * This class is handling the controller for the samples part
  * @author Florent Benoit
  */
 export class CreateProjectSamplesController {
+  $rootScope: ng.IScope;
```
Comment (akurinnoy) [inline dashboard/src/app/projects/create-project/samples/create-project-samples.controller.ts]: $rootScope: ng.IRootScopeService;

File: dashboard/src/app/projects/create-project/samples/create-project-samples.controller.ts [hunk 2]
```diff
@@ -9,110 +9,76 @@
...
+  templates: Array<any>;
+  selectedTemplateName: string = '';
 
   /**
    * Default constructor that is using resource
    * @ngInject for Dependency injection
    */
-  constructor($rootScope, cheAPI, $timeout, lodash) {
+  constructor($rootScope: ng.IScope, cheAPI: CheAPI) {
```
[the remark this answers is not in the document]
Comment (olexii4) [reply dashboard/src/app/projects/create-project/samples/create-project-samples.controller.ts]: done

**Comment by olexii4** [comment]
@slemeur  done

**Review by ashumilova — APPROVED** [review]

**Review by akurinnoy — APPROVED** [review]

**Review by slemeur — APPROVED** [review]
Wonderful !

**Pull request merged**